Sources/MQ/SourceCodeMeta/SourceCodeMeta.swift

```diff
@@ -106,7 +106,7 @@ extension SourceCodeMeta: CustomDebugStringConvertible {
 				.appending(
 					self.values
 						.reduce(
-							into: "",
```

Comment:
No difference except defining type explicitly, string literals can be resolved to many different types, in theory this could compile slightly faster (it does not matter overall). It is rather style change to be more explicit on types that actual change